src/v/config/configuration.cc

```diff
@@ -1620,6 +1603,44 @@ configuration::configuration()
       "write enabled",
       {.needs_restart = needs_restart::no, .visibility = visibility::user},
       24h)
+  , retention_local_is_advisory(
+      *this,
+      "retention_local_is_advisory",
+      "Allow log data to expand past local retention. When enabled, non-local "
+      "retention settings are used, and local retention settings are used to "
+      "inform data removal policies in low-disk space scenarios.",
+      {.needs_restart = needs_restart::no, .visibility = visibility::user},
+      false)
+  , retention_local_target_capacity_bytes(
```

Comment:
I wonder if it makes sense to consider `storage_space_alert_free_threshold_percent` et al at all as defaults, if we're looking for a default value

Reply:
Some related to this should be in the next PR that is going to introduce new config options for sizing.